src/Datadog.Trace/Agent/IApiRequest.cs

```diff
@@ -1,15 +1,12 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Datadog.Trace.Agent.MessagePack;
-using Datadog.Trace.Headers;
 
 namespace Datadog.Trace.Agent
 {
     internal interface IApiRequest
     {
+        void Cancel();
```

Comment:
I added this as a way to access cancellation tokens within each of the requests, but it's not actually used anywhere.
Thoughts?

Reply:
I'd say you can remove it for now since it won't be used and we haven't considered cancelling in-flight requests to the trace agent